openlibrary/templates/search/search_bar.html

```diff
@@ -0,0 +1 @@
+<input type="text" class="search-input" name="q" size="100" /><input type="submit" class="search-btn" value="$_('Search')"/>
```

Comment:
add a newline at the end of the file
```suggestion
<input type="text" class="search-input" name="q" size="100" /><input type="submit" class="search-btn" value="$_('Search')"/>

```